Gemfile.lock

```diff
@@ -92,26 +95,26 @@ GEM
       actionpack (>= 3.0)
       activesupport (>= 3.0)
       sprockets (>= 2.8, < 4.0)
-    thor (0.19.1)
-    thread_safe (0.3.5)
+    thor (0.20.0)
+    thread_safe (0.3.6)
     tilt (1.4.1)
-    tzinfo (1.2.2)
+    tzinfo (1.2.4)
       thread_safe (~> 0.1)
-    uglifier (3.0.0)
+    uglifier (3.2.0)
       execjs (>= 0.3.0, < 3)
-    unicode-display_width (1.0.3)
+    unicode-display_width (1.3.0)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
   cassandra-driver (~> 3.0)
   rails (= 4.1.14.2)
```

Comment:
This seems to not satisfy the reports for CVE-2016-6316 and CVE-2015-7581.
No wonder however, as we pin the rails version in `Gemfile`: https://github.com/mesosphere/tweeter/blob/master/Gemfile#L4 which should be changed and committed as well